.github/actions/retest-action/action.yml

```diff
@@ -0,0 +1,11 @@
+name: 'Re-Test'
+description: 'Re-Runs the last workflow for a PR'
```

Comment:
Let's update this comment to reflect that the new behavior will specifically rerun failing jobs.